dbflute-runtime/src/main/java/org/dbflute/cbean/ckey/ConditionKey.java

```diff
@@ -449,14 +450,21 @@ protected ColumnRealName resolveCompoundColumn(ColumnRealName baseRealName, Cond
         }
         final List<SpecifiedColumn> compoundColumnList = option.getCompoundColumnList();
         final List<ColumnRealName> realNameList = new ArrayList<ColumnRealName>();
-        realNameList.add(baseRealName); // already cipher
+        realNameList.add(resolveCompoundColumnOption(option, baseRealName));        
         for (SpecifiedColumn specifiedColumn : compoundColumnList) {
-            realNameList.add(doResolveCompoundColumn(option, specifiedColumn));
+            realNameList.add(resolveCompoundColumnOption(option, doResolveCompoundColumn(option, specifiedColumn)));
         }
         final OnQueryStringConnector stringConnector = option.getStringConnector();
         final String connected = stringConnector.connect(realNameList.toArray());
         return ColumnRealName.create(null, new ColumnSqlName(connected));
     }
+    
+    protected ColumnRealName resolveCompoundColumnOption(ConditionOption option, ColumnRealName columnRealName) {
```

Comment:
Thanks.
The two methods are bad balance, 
resolveCompoundColumnOption() and doResolveCompoundColumn().

The doResolveCompoundColumn() is already too abstract name in the now
so rename to doResolveCompoundColumnCipher() that the name has purpose.
(Existing methods can be adjusted when adding new methods)
And keep "already cipher" comment to tell someone why first column is not cipher target.

And a nested method has prefix 'do' like doResolveCompoundColumn()
so rename to doResolveCompoundColumnOption().

Reply:
> Thanks.
> The two methods are bad balance,
> resolveCompoundColumnOption() and doResolveCompoundColumn().
> 
> The doResolveCompoundColumn() is already too abstract name in the now
> so rename to doResolveCompoundColumnCipher() that the name has purpose.
> (Existing methods can be adjusted when adding new methods)
> And keep "already cipher" comment to tell someone why first column is not cipher target.
> 
> And a nested method has prefix 'do' like doResolveCompoundColumn()
> so rename to doResolveCompoundColumnOption().

thank you.

Reply:
renamed as you supposed.

https://github.com/dbflute/dbflute-core/pull/54/commits/a2cdd3a82193ced0da4cbb0b6bf12387f79ca97d